matrix/recognition/v1/recognition_service.proto

```diff
@@ -15,15 +15,15 @@
 
 syntax = "proto3";
 
-package matrixlabs.vision.v1;
+package matrix.recognition.v1;
 
-import "matrixlabs/vision/v1/vision_service.proto";
-import "matrixlabs/vision/v1/recognition.proto";
+import "matrix/vision/v1/vision_service.proto";
```

Comment:
@maciekrb would this be vision.proto versus vision_service.proto? Looks like we have "recognition.proto", just want to make sure the naming is aligned. Let me know.